machine_learning/management/commands/ml_smoothing.py

```diff
@@ -36,89 +22,97 @@
 NEIGH_MAX_CYC = 1
 NEIGH_RATE = 2
 NEIGH_CELL_ID = 3
-NEIGH_ABSOLUTE_CYCLE = 4
+NEIGH_ABSOLUTE_CYC = 4
 NEIGH_VALID_CYC = 5
 NEIGH_SIGN_GRID = 6
 NEIGH_VOLTAGE_GRID = 7
 NEIGH_CURRENT_GRID = 8
-NEIGH_TEMPERATURE_GRID = 9
+NEIGH_TMP_GRID = 9
```

Comment:
Really? 
this is a bad abreviation
Maybe NEIGH_T_GRID

Reply:
I thought `TMP` would be less ambiguous if it was in lowercase, or we could use `tmp` in lowercase and `T` in uppercase?